modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/utils.js

```diff
@@ -165,8 +165,67 @@ export function parseReassignments(node) {
 	return assignments;
 }
 
-export function parseNotifications(node) {
-	const notifications = {notificationTypes: [], recipients: []};
+function getRecipientsAttibutes(node, xmlDefinition, notificationIndex) {
```

Comment:
I believe getReceptionType is a better name for the function as it only returns me attributes of type ReceptionType. But I will wait for the relevant to make possible future changes only once.